CONTRIBUTING.md

```diff
@@ -54,16 +54,17 @@ others.
 Commits should generally be relevant, atomic, and have descriptions formatted
 in the following manner:
 
->  component: short description (<= 50 characters total)
+>  component: short description
```

Comment:
I would actually do `component: description (<= 50 characters total)` to keep that contained nicely within the format.

(My >50 chars callout wasn't mean as actual feedback, I was just joking around, this was fine the way it was...)